net/dcwifi/dcstad/Makefile

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (C) 2019 EWSI
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=dcstad
+PKG_VERSION:=1.0.0
+PKG_RELEASE:=1
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=COPYING
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/ewsi/$(PKG_NAME)/tar.gz/v$(PKG_VERSION)?
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_HASH:=3b146ea22bc5480d8264c5ea269831d25993673aa90a9e82dc2dc601a111da55
+PKG_MAINTAINER:=Carey Sonsino <careys@edgewaterwireless.com>
+
+PKG_FIXUP:=autoreconf
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/dcstad
+  SECTION:=net
+  CATEGORY:=Network
+  SUBMENU:=Routing and Redirection
+  TITLE:=Dual-Channel WiFi client daemon
+  URL:=https://www.edgewaterwireless.com
+  DEPENDS:= +libdcwsocket +libdcwproto
+  PKG_BUILD_DEPENDS:= libdcwsocket libdcwproto
+endef
+
+define Package/dcstad/description
+Implementation of the Dual-Channel WiFi client daemon
+endef
+
+export LINUX_ROOT=$(LINUX_DIR)
+export CROSS_COMPILE=$(TOOLCHAIN_DIR)/bin/$(TARGET_CROSS)
+export SUBDIRS=$(PKG_BUILD_DIR)
+
+define Build/Compile
```

Comment:
get rid of this section. Replace it with:

```
TARGET_CXXFLAGS += -std=c++98 -I$(STAGING_DIR)/usr/include
TARGET_LDFLAGS += -ldcwproto -ldcwsocket -lmrmfilterparser -luci
```

If you really need to specify the others as well, use MAKE_FLAGS +=